thoth/analyzer/cli.py

```diff
@@ -78,6 +79,7 @@ def print_command_result(click_ctx: click.core.Command,
         'analyzer_version': analyzer_version,
         'distribution': distro.info(),
         'arguments': _get_click_arguments(click_ctx),
+        'duration': int(duration) if duration is not None else None,
```

Comment:
is None good to be serialized into JSON? what abot 0 or -1 ?

Reply:
> is None good to be serialized into JSON? what abot 0 or -1 ?

@goern I just tried something like this:
```python
import json
obj = {'duration': None}
json.dumps(obj)
```
And the result was:
```json
{
 "duration": null
}
```
WDYT @fridex ?

Reply:
Yes, that's expected behavior.